content/en/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -103,12 +100,11 @@ This diagram shows the extension points in a Kubernetes system.
 
 If you are unsure where to start, this flowchart can help. Note that some solutions may involve several types of extensions.
 
-
-<img src="https://docs.google.com/drawings/d/e/2PACX-1vRWXNNIVWFDqzDY0CsKZJY3AR8sDeFDXItdc5awYxVH8s0OLherMlEPVUpxPIB1CSUu7GPk7B2fEnzM/pub?w=1440&h=1080">
-
 <!-- image source drawing: https://docs.google.com/drawings/d/1sdviU6lDz4BpnzJNHfNpQrqI9F19QZ07KnhnxVrp2yg/edit -->
+<img src="/docs/concepts/extend-kubernetes/flowchart.png">
```

Comment:
nit: We should use Markdown image references if we can, and set alt attributes.

Reply:
revised